x-pack/plugins/data_visualizer/public/application/index_data_visualizer/search_strategy/requests/overall_stats.ts

```diff
@@ -196,6 +196,7 @@ export const checkNonAggregatableFieldExistsRequest = (
     index,
     size,
     body: searchBody,
+    track_total_hits: 1,
```

Comment:
Does `track_total_hits` need to be set when enabling search session support? Just wondering why this needed to be set. Also we normally set this to be `true` rather than `1`.